extmod/modselect.c

```diff
@@ -107,6 +268,69 @@ STATIC mp_uint_t poll_map_poll(mp_map_t *poll_map, size_t *rwx_num) {
     return n_ready;
 }
 
+STATIC mp_uint_t poll_group_poll_all(poll_group_t *poll_group, size_t *rwx_num, mp_uint_t timeout) {
+    mp_uint_t start_ticks = mp_hal_ticks_ms();
+
+    #if MICROPY_PY_SELECT_POSIX_OPTIMISATIONS
+
+    for (;;) {
+        MP_THREAD_GIL_EXIT();
+
+        // Compute the timeout.
+        int t = 1;
```

Comment:
This `1` is potentially an important tuning parameter and should be a defined constant.

Reply:
This has been made a macro (private to this file, but in the future could be made public and configurable).